tests/spec/mesos_slave/mesos_slave_spec.rb

```diff
@@ -4,6 +4,30 @@
   it { should be_installed }
```

Comment:
should we add a test here to ensure the master service is not running on the slave?